internal_worker.go

```diff
@@ -97,9 +103,11 @@ type (
 		// Defines how many concurrent poll requests for the task list by this worker.
 		ConcurrentPollRoutineSize int
 
-		// Defines how many executions for task list by this worker.
-		// TODO: In future we want to separate the activity executions as they take longer than polls.
-		// ConcurrentExecutionRoutineSize int
+		// Defines how many concurrent executions for task list by this worker.
+		ConcurrentActivityExecutionSize int
+
+		// Defines rate limiting on number of activity tasks that can be executed per second.
+		MaxActivityExecutionRatePerSecond int
```

Comment:
cool.. my rule on this one is - any config that's based off time should have units as the suffix or it won't make sense to the reader :)